app/src/main/java/com/mybus/service/GeocodingService.java

```diff
@@ -7,27 +7,30 @@
 import android.util.Log;
 
 import com.google.android.gms.maps.model.LatLng;
+import com.mybus.location.OnAddressGeocodingCompleteCallback;
+import com.mybus.location.OnLocationGeocodingCompleteCallback;
 
 import java.io.IOException;
 import java.util.List;
 import java.util.Locale;
 
-public class LocationGeocoding {
+public class GeocodingService {
```

Comment:
For all the services we should extract the interface and interact with the service through it. This will make it easier to change the concrete implementation providing loose coupling.